awx/main/tasks.py

```diff
@@ -1179,11 +1179,17 @@ def final_run_hook(self, instance, status, private_data_dir, fact_modification_t
         instance.log_lifecycle("finalize_run")
         job_profiling_dir = os.path.join(private_data_dir, 'artifacts/playbook_profiling')
         awx_profiling_dir = '/var/log/tower/playbook_profiling/'
+        collections_info = os.path.join(private_data_dir, 'artifacts/', 'collections.json')
+
         if not os.path.exists(awx_profiling_dir):
             os.mkdir(awx_profiling_dir)
         if os.path.isdir(job_profiling_dir):
             shutil.copytree(job_profiling_dir, os.path.join(awx_profiling_dir, str(instance.pk)))
-
+        if os.path.exists(collections_info):
+            with open(collections_info) as ee_json_info:
+                ee_collections_info = ee_json_info.read()
```

Comment:
I _think_ we might need a `json.loads(ee_json_info.read())` here. In the demo you gave it "looked" like the data might be stored as a string in the database. The reason this works is that a top-level string is considered valid json in some circles and postgres is in that circle.

Below is an example of what it looks like when valid json is stored in a `jsonb` field vs. a string being stored.

https://asciinema.org/a/S451G5hKPoHMuywzm9QMxNtJM